1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,26 @@
+// - Create a `<p>` element.
+// - Create a `<select>` element.
+// - Iterate over the options array and create an `<option>` element in the `<select>` for each element.
+// - You'll need to turn the paragraph into an array to iterate over it. You can use the `.split(" ")` method here.
+// - Iterate over the array of words.
+// - For each word, create a `<span>` element and set the innerText to the word, plus a space - " ". Add this to the `<p>`. <<<???>>>
+// - Each `<span>` should have an eventListener that will listen for clicks.
+// - When clicked, we need to check the value of the `<select>` element using the `.value` property.
+// - We can then update the `background-color` property of the `<span>` with the value of the select - remember that the value "none" is a special case and we need to be handled differently.
+// - All of your HTML should go inside the `<div>` with the id **"content"**.
+
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+for (let a = 0; a < array.length; a++) {
+  let paragraph = document.createElement("p");
+  paragraph.innerHTML = paragraph;
+  paragraph.split("") = array;
```

Comment:
```suggestion
  paragraph.split(" ") = array;
```

Reply:
The argument of `.split()` should be `" "` as we want to split the paragraph based on the spaces between the words.